### ISSUE TITLE
[1.6-rc.8] Startup failing: logging package crashing on undefined color

### ISSUE THREAD

**Issue by brucejo75** [meteor/meteor#9225]
Failing on startup.

Updated to logging 1.1.19 and verified that code had changed in my build:

`if (isWin32 && typeof color === 'string' && color.slice(-6) !== 'Bright') {`

### Environment
- Windows 10 1703 (OS Build 15063.608)
- Meteor 1.6-rc.8

### Error Output
```
(node:3376) TypeError: Cannot read property 'slice' of undefined
    at platformColor (packages/logging.js:79:24)
    at Function.Log.format (packages/logging.js:300:23)
    at C:\tools\runners\run-log.js:104:29
    at Object.exports.noYieldsAllowed (C:\tools\utils\fiber-helpers.js:37:14)
    at RunLog.logAppOutput (C:\tools\runners\run-log.js:103:27)
    at C:\tools\runners\run-app.js:97:16
    at Promise.asyncApply (C:\tools\utils\eachline.js:19:20)
    at C:\Users\Bruce\AppData\Local\.meteor\packages\fourseven_scss\3.13.0\plugin.compileScssBatch.os\npm\node_modules\meteor\promise\node_modules\meteor-promise\fiber_pool.js:32:39
(node:3376) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    at emitWarning (internal/process/promises.js:78:15)
    at emitPendingUnhandledRejections (internal/process/promises.js:95:11)
    at runMicrotasksCallback (internal/process/next_tick.js:124:9)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by brucejo75** [comment]
Also, is there a way I can downgrade to logging 1.1.17 with rc.8?  I tried changing in the `versions` file but it upgrades on build 😕

**Comment by benjamn** [comment]
Is there anything in `.meteor\local\isopacks` in your app?

What happens if you `meteor update fourseven:scss` (3.13.0 is an old version)?

**Comment by benjamn** [comment]
https://github.com/meteor/meteor/pull/8728#issuecomment-336517159

**Comment by brucejo75** [comment]
@benjamn, i verified that `.meteor\local\build\programs\server\packages\logging.js` had the fixes in it.  Is that where the build gets the logging package?

**Comment by brucejo75** [comment]
fixed with 1.6-rc.9
